src/commands/api-mesh/__tests__/log-get.test.js

```diff
@@ -0,0 +1,186 @@
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+jest.mock('axios');
+jest.mock('@adobe/aio-lib-env');
+jest.mock('@adobe/aio-cli-lib-console');
+jest.mock('fs/promises');
+jest.mock('@adobe/aio-cli-lib-console', () => ({
+	init: jest.fn().mockResolvedValue(mockConsoleCLIInstance),
+	cleanStdOut: jest.fn(),
+}));
+jest.mock('@adobe/aio-lib-ims');
+jest.mock('../../../helpers', () => ({
+	initSdk: jest.fn().mockResolvedValue({}),
+	initRequestId: jest.fn().mockResolvedValue({}),
+}));
+
+jest.mock('../../../lib/devConsole');
+
+jest.mock('chalk', () => ({
+	red: jest.fn(text => text), // Return the input text without any color formatting
+	bold: jest.fn(text => text),
+	underline: {
+		blue: jest.fn(text => text),
+	},
+	bgYellow: jest.fn(text => text),
+}));
+
+const mockConsoleCLIInstance = {};
+const selectedOrg = { id: '1234', code: 'CODE1234@AdobeOrg', name: 'ORG01', type: 'entp' };
+const selectedProject = { id: '5678', title: 'Project01' };
+const selectedWorkspace = { id: '123456789', title: 'Workspace01' };
+
+const { writeFile } = require('fs/promises');
+const { initSdk } = require('../../../helpers');
+const FetchLogsCommand = require('../log-get');
+const { getLogsByRayId, getMeshId } = require('../../../lib/devConsole');
+const os = require('os');
+let logSpy = null;
+let errorLogSpy = null;
+let parseSpy = null;
+let platformSpy = null;
+
+const mockIgnoreCacheFlag = Promise.resolve(true);
+
+describe('FetchLogsCommand tests', () => {
+	beforeEach(() => {
+		initSdk.mockResolvedValue({
+			imsOrgId: selectedOrg.id,
+			projectId: selectedProject.id,
+			workspaceId: selectedWorkspace.id,
+			workspaceName: selectedWorkspace.title,
+		});
+
+		global.requestId = 'dummy_request_id';
+
+		logSpy = jest.spyOn(FetchLogsCommand.prototype, 'log');
+		errorLogSpy = jest.spyOn(FetchLogsCommand.prototype, 'error');
+		platformSpy = jest.spyOn(os, 'platform');
+
+		writeFile.mockResolvedValue(true);
+
+		parseSpy = jest.spyOn(FetchLogsCommand.prototype, 'parse');
+		parseSpy.mockResolvedValue({
+			args: { rayId: 'ray1' },
+			flags: {
+				ignoreCache: mockIgnoreCacheFlag,
+			},
+		});
+		getMeshId.mockResolvedValue('12345');
+		getLogsByRayId.mockResolvedValue({
+			'EventTimestampMs': 123456789,
+			'Exceptions': 'None',
+			'Logs': 'Log data',
+			'Outcome': 'Success',
+			'meshId': 'mesh1',
+			'rayId': 'ray1',
+			'URL': 'http://example.com',
+			'Request Method': 'GET',
+			'Response Status': 200,
+		});
+	});
+
+	afterEach(() => {
+		platformSpy.mockRestore();
+	});
+
+	test('snapshot FetchLogsCommand', () => {
+		expect(FetchLogsCommand.description).toMatchInlineSnapshot(
+			`"Get the Log of a given mesh by RayId"`,
+		);
+		expect(FetchLogsCommand.args).toMatchInlineSnapshot(`
+		        [
+		          {
+		            "description": "to fetch the log ",
```

Comment:
This might have to be changed based on the feedback from @jhadobe

Reply:
```suggestion
		            "description": "Fetch a single log by rayID ",
```